manuscript/reviewer_reply.Rmd

```diff
@@ -31,15 +31,19 @@ Line 41: I think this is generally more common in terrestrial ecology – in fis
 
 Line 62: I really like the recommendations / clarity in Box 1, and think these will be accessible to most ecologists
 
-Line 71-73: I’m re-reading this after commenting on data, and am now a little confused. The bird responses are annual snapshots during the breeding season? And I thought the environmental data you used was annual or quarterly, not monthly?
-https://www.mbr-pwrc.usgs.gov/bbs/grass/genintro.htm
+"Line 71-73: I’m re-reading this after commenting on data, and am now a little confused. The bird responses are annual snapshots during the breeding season? And I thought the environmental data you used was annual or quarterly, not monthly?
+https://www.mbr-pwrc.usgs.gov/bbs/grass/genintro.htm"
+
+--BBS data are generally viewed as annual data, even though the observations happen on a single day. The presence of an individual species is primarily a function of suitable habitat and, for many species, timing of migration. Environmental variables with a shorter temporal scale could likely improve models, such as incorporating cool weather days into the observation model, and would be worth pursuing in a different study. 
```

Comment:
I think we can be clearer here.

Reply:
@ethanwhite Any thoughts on how to make this part clearer (both in the manuscript and the response)? It's kind of subtle.

Reply:
@ethanwhite Actually, in the interest of avoiding merge conflicts, I'll just merge now and create a separate issue.